applications/nrf_desktop/src/modules/power_manager.c

```diff
@@ -262,7 +262,7 @@ static bool event_handler(const struct event_header *eh)
 		switch (event->state) {
 		case USB_STATE_POWERED:
 		case USB_STATE_ACTIVE:
-			k_delayed_work_cancel(&power_down_trigger);
```

Comment:
Ok. Cancels will not race with any resubmit - all operations are in sync with system workqueue.